zml/testing.zig

```diff
@@ -13,19 +13,17 @@ var _platform: ?zml.Platform = null;
 pub fn env() zml.Platform {
     if (!builtin.is_test) @compileError("Cannot use zml.testing.env outside of a test block");
     if (_platform == null) {
-        _test_compile_opts = .{
-            .sharding_enabled = true,
-        };
-
         var ctx = zml.Context.init() catch unreachable;
-        _platform = ctx.autoPlatform(.{}).withCompilationOptions(_test_compile_opts);
+        _platform = ctx.autoPlatform(.{}).withCompilationOptions(.{
+            .xla_dump_to = "/tmp/zml/tests/",
```

Comment:
We might want to consider an option to opt-out from module caching when we use the `xla_dump_to` because we don't want that for tests

Reply:
yes on providing the opt-out, but for tests it's pretty good to have the full test suite running in seconds when you iterate on a small part of the library